src/headlessMoneyHash.ts

```diff
@@ -470,6 +473,8 @@ export default class MoneyHashHeadless<TType extends IntentType> {
        * @returns { Elements }
        */
       create: ({ elementType, elementOptions }: ElementProps): Element => {
+        this.createdCardElements.push(elementType);
```

Comment:
Check mounted or not better

It can be created with mounting and will be the same issue